src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -511,4 +535,36 @@ private void importEntries(List<CitationRelationItem> entriesToImport, CitationF
 
         dialogService.notify(Localization.lang("Number of entries successfully imported") + ": " + entriesToImport.size());
     }
+
+    /**
+     * Function to open possible duplicate entries window to compare duplicate entries
+     *
+     * @param duplicateItem duplicate in the citation relations tab
+     */
+    private void openPossibleDuplicateEntriesWindow(CitationRelationItem duplicateItem) {
+        BibEntry localEntry = duplicateItem.localEntry();
+        BibEntry duplicateEntry = duplicateItem.entry();
+
+        MergeEntriesDialog dialog = new MergeEntriesDialog(localEntry, duplicateEntry, preferences);
+        dialog.setTitle(Localization.lang("Possible duplicate entries"));
+
+        Optional<EntriesMergeResult> mergeResultOpt = dialogService.showCustomDialogAndWait(dialog);
```

Comment:
Do you really need the `Opt` suffix? - You can name it `entriesMergeResult` and one line below use `mergeResult` if you like shorter variables.